examples/nlp/md/tweet-classification-using-tfdf.md

```diff
@@ -347,26 +347,4582 @@ model_2.fit(train_ds)
 
 <div class="k-default-codeblock">
 ```
-Starting reading the dataset
-77/77 [==============================] - ETA: 0s
-Dataset read in 0:00:15.844516
-Training model
-Model trained in 0:02:30.922245
-Compiling model
-77/77 [==============================] - 167s 2s/step
-Starting reading the dataset
-55/77 [====================>.........] - ETA: 0s
-Dataset read in 0:00:00.219258
-Training model
-Model trained in 0:00:00.289591
-Compiling model
-77/77 [==============================] - 1s 6ms/step
+Reading training dataset...
+Training dataset read in 0:00:06.473683. Found 6852 examples.
+Training model...
+Model trained in 0:00:41.461477
+Compiling model...
 
-<keras.callbacks.History at 0x7f453f9349d0>
+[INFO kernel.cc:1176] Loading model from path /tmp/tmpsp7fmsyk/model/ with prefix 20297ba36a694abd
+[INFO abstract_model.cc:1248] Engine "GradientBoostedTreesQuickScorerExtended" built
+[INFO kernel.cc:1022] Use fast generic engine
 
+WARNING:tensorflow:AutoGraph could not transform <function simple_ml_inference_op_with_handle at 0x7fe0aaefb5b0> and will run it as-is.
```

Comment:
There are lots of WARNING and INFO logs in the file. Please remove them, they're noise.